test/libponyrt/ds/list.cc

```diff
@@ -159,7 +159,7 @@ TEST_F(ListTest, ListIndexAbsolute)
   testlist_t* list = testlist_append(NULL, &e1);
   list = testlist_append(list, &e2);
 
-  testlist_t* n = testlist_index(list, i);
+  testlist_t* n = testlist_index(list, (ssize_t)i);
```

Comment:
`i` is declared as a `size_t` a few lines above this - could we just declare it as a `ssize_t` instead of converting it here?

Reply:
I suspect `i` gets used in more places where it makes more sense for it to be unsigned and then having a signed type that we have to convert to unsigned is more problematic.

Reply:
No, it doesn't seem to be. It's a local variable here in the scope of this block from lines 153-166, and doesn't seem to be used anywhere else in this block

Reply:
True! Fixed-up the `test: ...` commit.